pydis_site/apps/content/resources/guides/pydis-guides/contributing/sir-lancebot.md

```diff
@@ -121,13 +111,4 @@ After installing project dependencies use the poetry command `poetry run task st
 $ poetry run task start
 ```
 
----
-
-# Working with Git
```

Comment:
I do think it's worth having a mention of git here and in the other pages as well, even if it's just linking to the walkthrough/subpages. People can reach these pages in different ways, and as far as guides and walkthroughs go I think redundancy can be useful.

Reply:
I kind of disagree. Our set-up guides are very long. I think keeping them focused will make them less intimidating and implicitly communicate that people should look for non-set up info elsewhere.

What do you think about a little blurb at the top that says something like "This page is for setting up the env. For info on how to get started.." and points to ./contributing?

Reply:
What you say makes sense, but in practice from the way people use these guides I think mentioning the same thing in different places helps. Even if we don't explain how to use git all over again, saying that the repo should be cloned as the first step (with a link to ./contributing) makes sense to me here. It also makes sense in terms of continuity since later in the page it says "You now have both the bot's code and a server to run it on". A person going over this guide might think "wait, when did I get the code?" and then might even proceed to download it without git.

Reply:
I added reminders to fork and clone at the tops of the set-up guides and "Next steps" sections at the bottoms that link to the walk-through and contributing guidelines.